apps/point_cloud_editor/CMakeLists.txt

```diff
@@ -1,33 +1,15 @@
 set(SUBSUBSYS_NAME point_cloud_editor)
 set(SUBSUBSYS_DESC "Point Cloud Editor - Simple editor for 3D point clouds")
 set(SUBSUBSYS_DEPS common filters io apps)
-
-# QT5 Found?
-if(NOT Qt5_FOUND)
-  set(DEFAULT AUTO_OFF)
-  set(REASON "Qt5 was not found.")
-elseif(NOT ${DEFAULT} STREQUAL "AUTO_OFF")
-  set(DEFAULT TRUE)
-  set(REASON)
-endif()
-
-# Find OpenGL
-if(NOT OPENGL_FOUND)
-  set(DEFAULT AUTO_OFF)
-  set(REASON "OpenGL was not found.")
-elseif(NOT ${DEFAULT} STREQUAL "AUTO_OFF")
-  set(DEFAULT TRUE)
-  set(REASON)
-endif()
-
+set(SUBSUBSYS_EXT_DEPS vtk Qt5 OpenGL)
```

Comment:
Do OpenGL and OPENGL both work?

Reply:
It should, as the dependencies gets upper-cased:
https://github.com/PointCloudLibrary/pcl/blob/f8a48ddd61141601c6b718a4d15d9040ca0e1662/cmake/pcl_targets.cmake#L114-L122
and here:
https://github.com/PointCloudLibrary/pcl/blob/f8a48ddd61141601c6b718a4d15d9040ca0e1662/cmake/pcl_targets.cmake#L170-L177